lib/state-controller.js

```diff
@@ -3,70 +3,63 @@
 const fs = require('fs');
 const os = require('os');
 const querystring = require('querystring');
+const https = require('https');
 
 const Client = require('./client.js');
 const KiteError = require('./kite-error');
 const utils = require('./utils.js');
+const NO_SUPPORT = require('./support/no-support');
+const {STATES} = require('./constants');
 
 const StateController = {
   client: new Client('127.0.0.1', 46624, '', false),
 
-  STATES: {
-    UNSUPPORTED: 0,
-    UNINSTALLED: 1,
-    INSTALLED: 2,
-    RUNNING: 3,
-    REACHABLE: 4,
-    AUTHENTICATED: 5,
-    WHITELISTED: 6,
-  },
+  STATES,
```

Comment:
Two things here: 
- I moved STATES out of the state controller to avoid cycling references (`StateController` requires `OSXSupport` and `OSXSupport` needs to access `STATES` so it requires the `StateController`). By moving it in a third file it can be required by both without any issue.
- It's still stored in the `StateController` so that the existing code that relies on `StateController.STATES` does not break with that change.